presto-product-tests/conf/docker/common/kerberos.yml

```diff
@@ -13,4 +13,4 @@ services:
   application-runner:
     image: 'teradatalabs/cdh5-hive-kerberized:${DOCKER_IMAGES_VERSION}'
     volumes:
-      - ../../../conf/tempto/tempto-configuration-for-docker-kerberos.yaml:/docker/volumes/tempto/tempto-configuration-local.yaml
+    - ${TEMPTO_CONFIG_KERBEROS_YAML}:/docker/volumes/tempto/tempto-configuration-local.yaml
```

Comment:
This line can be removed, as the entry in standard.yml will be parametrized with TEMPTO_CONFIG_YAML.